framework/include/chaincontrols/LimitChainControl.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ChainControl.h"
+
+/**
+ * Limits a control value by a range.
+ */
+class LimitChainControl : public ChainControl
+{
+public:
+  static InputParameters validParams();
+
+  LimitChainControl(const InputParameters & parameters);
+
+  virtual void execute() override;
+
+protected:
+  /// Minimum value to apply to control data
+  const Real _min_value;
+  /// Maximum value to apply to control data
```

Comment:
```suggestion
  /// Maximum value to restrict control data
```

Reply:
Changed wording to "Lower/upper bound to apply to"